test/fixtures/poro.rb

```diff
@@ -46,6 +46,13 @@ def arguments_passed_in?
   end
 end
 
+class Person < Model
```

Comment:
can you move these to your test file?

We really don't like the poros file, as it makes deugging tests a little more convoluted than it should be